packages/nimble-components/src/select/index.ts

```diff
@@ -49,6 +69,122 @@ export class Select extends FoundationSelect implements ErrorPattern {
     @observable
     public hasOverflow = false;
 
+    /**
+     * @internal
+     */
+    @observable
+    public filteredOptions: ListboxOption[] = [];
+
+    /**
+     * @internal
+     */
+    @observable
+    public filter = '';
+
+    /**
+     * @internal
+     */
+    @observable
+    public scrollbarIsVisible = false;
+
+    private committedSelectedOption: ListboxOption | undefined = undefined;
+
+    public constructor() {
+        super();
+        this.addEventListener('change', this.changeValueHandler);
+        this.addEventListener('focusout', this.focusoutHander);
+    }
+
+    public override connectedCallback(): void {
+        super.connectedCallback();
+    }
```

Comment:
It doesn't seem like we need this `connectedCallback` if all it does is call the super class.